### PR TITLE
Create Get-SPOListDesign.md

### PR DESCRIPTION
**Pull request by SaladiHarini**
Adding a new cmdlet to support the Custom List Templates feature!

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Get-SPOListDesign [[-Identity] <SPOListDesignPipeBind>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Gets details about list designs that are on the SharePoint tenant. You can specify an ID of a specific list design to retrieve. If there are no parameters listed, details about all list designs are listed. 
+
+## EXAMPLES
+
+### Example 1
+This example and sample response show how to get list design details.
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md:25]:
```suggestion
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+Get-SPOListDesign 44252d09-62c4-4913-9eb0-a2a8b8d7f863 
+
+Id: 44252d09-62c4-4913-9eb0-a2a8b8d7f863  
+Title: Contoso customer tracking  
+SiteScriptIds: {1306913c-8463-42ca-bd63-efad0fcdbba4}  
+Description: Tracks key customer data in a list 
+ListColor: Orange 
+ListIcon: BullseyeTarget
+```
+
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md:37]:
```suggestion

This example and sample response show how to get list design details. 

```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md:56]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+## INPUTS
+
+### Microsoft.Online.SharePoint.PowerShell.SPOListDesignPipeBind
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md:60]:
```suggestion
Microsoft.Online.SharePoint.PowerShell.SPOListDesignPipeBind
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+## OUTPUTS
+
+### System.Object
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md:64]:
```suggestion
System.Object

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @SaladiHarini, kindly commit the suggested changes. Many thanks.

**Comment by yogkumgit** [comment]
@SaladiHarini Please implement the suggested changes. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@SaladiHarini Please implement the suggested changes. Thanks

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+## DESCRIPTION
+Gets details about list designs that are on the SharePoint tenant. You can specify an ID of a specific list design to retrieve. If there are no parameters listed, details about all list designs are listed. 
+
+## EXAMPLES
+
+### Example 1
+
+```powershell
+Get-SPOListDesign 44252d09-62c4-4913-9eb0-a2a8b8d7f863
```
[the remark this answers is not in the document]
Comment (SaladiHarini) [reply sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md]: Added!

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
+---
+external help file: Microsoft.Online.SharePoint.PowerShell.dll-Help.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version:
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md:4]:
```suggestion
online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spolistdesign
```

File: sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+schema: 2.0.0
+author: SaladiHarini
+ms.author: hasaladi
+---
```
Comment (get-itips) [inline sharepoint/sharepoint-ps/sharepoint-online/Get-SPOListDesign.md:8]:
```suggestion
title: Get-SPOListDesign
---
```

**Comment by get-itips** [comment]
> @get-itips I've made the requested changes by adding author metadata + adding the -Identity param to the example. Please review and let me know if any other changes needed here. At this point, this build is not failing, so phew!

Thank you, please see last requested changes

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward Please review. Thanks

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
close > reopen

**Pull request merged**